paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetCompose.kt

```diff
@@ -19,7 +19,7 @@ import com.stripe.android.utils.rememberActivityOrNull
 @ExperimentalCustomerSheetApi
 @Composable
 fun rememberCustomerSheet(
-    configuration: CustomerSheet.Configuration = CustomerSheet.Configuration(),
+    configuration: CustomerSheet.Configuration,
```

Comment:
I’m not sure what to think about this. Should we relax this to a nullable config and create a default configuration internally, with the application label as the merchant name?

Reply:
Totally possible. This is what we do for PaymentSheet.

If everything else in this sounds good to you, I think we should definitely do that and ship it.

Reply:
Let’s do it!

Reply:
Chatted offline. Slightly more inclined to leave it as not optional since the non compose way to instantiate CustomerSheet already requires a config.